src/controller/tests/data_model/TestRead.cpp

```diff
@@ -4715,4 +4758,9 @@ int TestReadInteractionTest()
     return chip::ExecuteTestsWithContext<TestContext>(&sSuite);
 }
 
+const nlTest * TestFoo()
```

Comment:
What is this and why is it here?

Reply:
That was a bit of debugging I was doing, and I committed by mistake. Reverted, sorry for the inconvenience.